darwin/importer/importer.py

```diff
@@ -278,18 +284,42 @@ def _resolve_annotation_classes(
     return local_classes_not_in_dataset, local_classes_not_in_team
 
 
-def _get_team_properties_annotation_lookup(client, team_slug):
+def _get_team_properties_annotation_lookup(
+    client: "Client", team_slug: str
+) -> Tuple[Dict[Tuple[str, Optional[int]], FullProperty], Dict[str, FullProperty]]:
+    """Returns two lookup dictionaries for team properties:
+     - team_properties_annotation_lookup: (property-name, annotation_class_id): FullProperty object
+     - team_item_properties_lookup: property-name: FullProperty object
```

Comment:
nit: would be nice to have consistency in naming, makes it easier to understand: `team_item_properties_lookup` and  `team_annotation_properties_lookup`

Reply:
Unfortunately I think `team_properties_annotation_lookup` has to be named this way. If it was called `team_annotation_properties_lookup`, this would imply it only contains data for annotation-level properties. However, it contains data for annotation-level and section-level properties. The naming convention for variables will be addressed in the DAR-4295 refactor